### ISSUE TITLE
2019 Trawl Survey Data

### ISSUE THREAD

**Issue by NickShafferNOAA** [nwfsc-fram/boatnet#1426]
2019 Trawl data needs to be copied over to the warehouse, most likely via Pentaho.

**Comment by BHHorness-NOAA** [comment]
After data copy to staging db dw, I discovered that zero specimen ids remained for the otosag, stomach, tissue, fin clip, and ovary specimen ids.  Nulled those out by hand in the staging individual_fact table.  Note that an anomalous otosag_id remains (11725697); did not troubleshoot.  Also, Kate Richerson discovered an anomalous weight value for goblet sponge in the 2019 trawl survey permit catch report I delivered to her.  Looked into it - the entered weight for the sponge was valid, but it was part of a mix that had one very large basket weight (843.6kg). In consultation with the field crew (Draper, Buchanan) and others (Keller, Frey) decided that the weight was simply in error and replaced the weight with the average of the other entered baskets for the Mix (25.04kg) by hand.  Since there was no subsample basket marked, I assumed that it hadn't been weighed unsorted and added in a new basket of weight equaling the summed mix species weights (12.04kg).  This could have just as easily been one of the existing entered mix baskets left unmarked; the smallest basket weight (and so closest to the summed species weight) was 14.95kg which would have worked. But I assumed it was more likely that the sub basket was sorted before it was weighed unsorted and any error incurred by adding a new basket vs marking an existing basket was minimal in this very large invert mix.  I used the dw_catch_datatransfer_2018_vw view in fram_central to determine the expanded weights and counts for the warehouse and computed the cpues from them - all entered into the staging individual_fact table manually.
